test/interactions/interactionTests.ts

```diff
@@ -0,0 +1,110 @@
+///<reference path="../testReference.ts" />
+
+var assert = chai.assert;
+
+describe("Interactions", () => {
+  describe("Interaction", () => {
```

Comment:
I was going to say that we didn't need this inner `describe()` at all, since anything on the top level (directly under `describe("Interactions")`) could probably be safely assumed to test the root class.